libraries/Microsoft.Bot.Builder.Azure.Blobs/BlobsTranscriptStore.cs

```diff
@@ -55,6 +74,8 @@ public BlobsTranscriptStore(string dataConnectionString, string containerName, J
                 throw new ArgumentNullException(nameof(containerName));
             }
 
+            _storageTransferOptions = storageTransferOptions;
```

Comment:
If blob complains about null transfer options, should we check for null and default here?